ansible/roles/test/tasks/vlan_cleanup.yml

```diff
@@ -0,0 +1,23 @@
+- name: Restore config_db.json
+  shell: mv /etc/sonic/config_db_bak.json /etc/sonic/config_db.json
+  become: true
+
+- name: Reboot is required for config_db.json change
```

Comment:
Same question as previous one,  is this reboot avoidable by using  "config reload config_db_back.json"?